composer.json

```diff
@@ -2,15 +2,15 @@
     "name": "magento/magento-cloud-patches",
     "description": "Provides critical fixes for Magento 2 Enterprise Edition",
     "type": "magento2-component",
-    "version": "1.0.1",
+    "version": "1.0.2",
```

Comment:
I guess better to change version exactly in the branch for preparing new release, not in the bug fix (just for future)

Reply:
Agree, but that would require some additional time and efforts to process 1 more PR